README.md

```diff
@@ -13,10 +13,10 @@ TimeZones.jl
 
 ## Features
 
-* A new time zone aware TimeType: ZonedDateTime
+* A new time-zone aware TimeType: ZonedDateTime
```

Comment:
I try to stick to having "time zone" be two words: https://en.wikipedia.org/wiki/Time_zone
```suggestion
* A new time zone aware TimeType: ZonedDateTime
```